stub-packages/merge-styles/src/DeepPartial.ts

```diff
@@ -0,0 +1 @@
+export * from '@uifabric/merge-styles/lib/DeepPartial';
```

Comment:
DeepPartial and other exports not listed in v8 merge-styles package.json/exports should not be included here.
https://github.com/microsoft/fluentui/blob/master/packages/merge-styles/package.json#L34-L70
![image](https://user-images.githubusercontent.com/1434956/187997714-21b3846d-ff66-4628-83fd-cdf013ee9bac.png)